e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/your-buyer/change-your-answers/change-your-answers-change-alternative-currency.spec.js

```diff
@@ -152,6 +153,30 @@ context('Insurance - Check your answers - Your buyer - Alternative currency - As
         cy.checkText(row.value(), expected);
       });
 
+      it(`should render the ${TOTAL_AMOUNT_OVERDUE} answer with the alternative currency on the outstanding or overdue payments page`, () => {
```

Comment:
worth using a describe here as is quite long? same with one below?

Reply:
I think it would overcomplicate it.

Simplified current approach:

```js
it(`should render the new answer for ${TOTAL_AMOUNT_OVERDUE}`, () => {
});

it(`should render the new answer for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
});

it(`should render the ${TOTAL_AMOUNT_OVERDUE} answer with the alternative currency on the outstanding or overdue payments page`, () => {
});

it(`should render the ${TOTAL_OUTSTANDING_PAYMENTS} answer with the alternative currency on the outstanding or overdue payments page`, () => {
});
```

Simplified proposed approach:

```js
describe('on the summary list', () => {
  beforEach(() => {
    cy.navigateToUrl(url);
  });

  it(`should render the new answer for ${TOTAL_AMOUNT_OVERDUE}`, () => {
  });

  it(`should render the new answer for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
  });
});

describe('on the outstanding or overdue payments page', () => {
  beforEach(() => {
    cy.navigateToUrl(otherUrl);
  });

  it(`should render the ${TOTAL_AMOUNT_OVERDUE} answer with the alternative currency`, () => {
  });

  it(`should render the ${TOTAL_OUTSTANDING_PAYMENTS} answer with the alternative currency`, () => {
  });
});
```